### DISCUSSION TITLE
Angehängte Dateien für Gäste abrufbar?

### DISCUSSION THREAD

**Discussion by git-tec** [thorsten/phpMyFAQ#2505] [Q&A] [answered]
Guten Morgen zusammen,

bekomme ich es irgendwie hin, dass Anhänge auch Gästen zu Verfügung stehen? aktuell werden Sie zwar als Text angezeigt, aber nicht als Download. In der Konfiguration habe ich "Erlaube Downloads von Gästen" aktiviert, aber scheinbar hat das keine Auswirkungen.

Installiert ist derzeit phpMyFAQ 3.1.14

**Comment by thorsten** [comment]
Das ist ein Seiteneffekt von #2497 und ist somit in der 3.1.15 gefixt.

**Reply by git-tec** [reply]
Also den Fix von #2497 (https://github.com/thorsten/phpMyFAQ/commit/0a904c15d23719f484c8be126245726d1a91c7a6) hatte ich händisch bereits davor eingespielt. Bilder und Formatierungen gehen wieder, aber Anhänge als Gast und als eingeloggter User aber nicht.

**Reply by thorsten** [reply]
Dann schau ich nochmal nach.

**Reply by thorsten** [reply, marked as answer]
This should fix it: https://github.com/thorsten/phpMyFAQ/commit/c4080d95a576e6f0e1e56fd8e6c5480136ac0a14
